lib/features/qr_scanner/presentation/camera_screen.dart

```diff
@@ -82,4 +96,101 @@ class _CameraScreenState extends State<CameraScreen> {
       },
     );
   }
+
+  Widget _buildPermissionDialog({bool isSettings = false}) {
+    return Dialog(
+      backgroundColor: Colors.white,
+      surfaceTintColor: Colors.white,
+      child: Stack(
+        children: [
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 24),
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                const SizedBox(height: 20),
```

Comment:
Why don't you put the first and last sizeboxes not also a padding? :)

Reply:
For me it would be more clear that it's part of padding in this way :) 
But both are fine indeed! Same result ;)